src/backend/auth/admin_access_endpoints.py

```
@@ -0,0 +1,243 @@
+
+
+@admin_access_router.post("/users/{user_id}/reassign")
+def reassign_user(user_id: str, body: _Reassign, request: Request,
+                  admin: dict = Depends(require_admin)):
+    """Move a user from one team org to another. Bumps token_version so the user's
+    stale (wrong-org) token dies and their next request forces a fresh login into
+    a correct-org token."""
+    try:
+        _orgs.reassign_user_org(user_id, body.from_org_id, body.to_org_id, body.org_role)
+    except (psycopg.errors.ForeignKeyViolation, psycopg.errors.InvalidTextRepresentation, ValueError):
+        raise HTTPException(400, "Invalid ids or non-team target org")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)